contrib/python/src/python/pants/contrib/python/checks/tasks/python_eval.py

```diff
@@ -104,6 +104,9 @@ def _compile_target(self, target):
       if isinstance(target, PythonBinary):
         source = 'entry_point {}'.format(target.entry_point)
         components = target.entry_point.rsplit(':', 1)
+        if not all([x.strip() for x in components]):
+          raise TaskError('Invalid entry point {} for target {}'.format(
```

Comment:
Yeah, the error is that you'll get "from <blank> import something" or "from something import <blank>" or even just "import <blank>". All of these will cause a weird error when the pex is run, better to catch it now.  I happened to notice this while debugging the `__init__.py` issue (because the root-level `__init__.py` tickles this.